.changeset/shy-ears-report.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sveltejs/kit': minor
+---
+
+feat: support caching of responses with `Vary` header (except for `Vary: *`)
+
+fix: include `Vary: Accept` header to fix browser caching of adjacent pages and endpoints
```

Comment:
you should have two changeset files if you want to have two changeset entries rather than putting them both in the same file or else it will mess up the changelog formatting